### PR TITLE
created Nancy.java

### PR DESCRIPTION
**Pull request by nancyp2510**

### CODE REVIEW

**Comment by alisasoki** [comment]
Do we need to have descriptions for the project?

**Review by alisasoki — APPROVED** [review]
Good job.

**Review by alisasoki — APPROVED** [review]
Good job!

File: src/Nancy.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class Nancy {
+    // Nancy create this file
+
+    // On local nancy branch
+
+    // uddating Nancy file
+}
```
Comment (omidsharifi1) [inline src/Nancy.java:7]: Looks good to me.

**Review by omidsharifi1 — APPROVED** [review]
Just need some title to make it clear to read.

**Comment by nancyp2510** [comment]
thank you guys. Omid : I thought i added title as " created nancy.java" is title? Gotta write something different?

**Comment by omidsharifi1** [comment]
Sorry, I meant description.

**Pull request merged**